content/2_Bronze/Intro_Sets.mdx

```diff
@@ -755,3 +755,203 @@ Some of these problems can be solved by sorting alone, though sets or maps could
+    <Quiz.Answer>
+      $-3$
+      <Quiz.Explanation>
+        The element $3$ isn't contained in $s$, so the output should be $0$, since ```s.count(x)``` is supposed to output $1$ when the $x$ is present in the set and $0$ when $x$ isn't present in the set.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    Which of the following is the correct output of this code?
+	```cpp
+	#include <bits/stdc++.h>
+	using namespace std;
+	int main()
+	{
+		vector<int> a = {4, 1, 2, 1, 3, 2, 3, 1, 3, 1};
+		set<int> s;
+		for(int x: a) s.insert(x);
+		cout << s.size() << endl;
+	}
+```
+    <Quiz.Answer>
+      $1$
+      <Quiz.Explanation>
+        There are a total of $4$ unique numbers inserted into $s$, so the answer is $4$.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      $4$
+      <Quiz.Explanation>
+        There are a total of $4$ unique numbers inserted into $s$, so the answer is $4$.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $7$
+      <Quiz.Explanation>
+        There are a total of $4$ unique numbers inserted into $s$, so the answer is $4$.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $10$
+      <Quiz.Explanation>
+        There are a total of $4$ unique numbers inserted into $s$, so the answer is $4$.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    What is the time complexity of insertions, deletions, and removals in an ordered map?
+    <Quiz.Answer>
+      $\mathcal{O}(1)$
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $\mathcal{O}(N)$
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $\mathcal{O}(N \log N)$
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      $\mathcal{O}(\log N)$
+      <Quiz.Explanation>
+				Correct.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    Which of the following is the correct output of this code?
+	```cpp
+	#include <bits/stdc++.h>
+	using namespace std;
+	int main()
+	{
+		map<int, int> m;
+		m[1] = 5;
+		m[2] = 9;
+		m[3] = 7;
+		m[4] = 3;
+		cout << m[2] << endl;
+	}
+	```
+    <Quiz.Answer>
+      $5$
+      <Quiz.Explanation>
+        The value associated with the key $2$ is $9$.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      $9$
+      <Quiz.Explanation>
+        The value associated with the key $2$ is $9$.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $2$
+      <Quiz.Explanation>
+        The value associated with the key $2$ is $9$.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $0$
+      <Quiz.Explanation>
+        The value associated with the key $2$ is $9$.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    Which of the following is the correct output of this code?
+	```cpp
+	#include <bits/stdc++.h>
+	using namespace std;
+	int main()
+	{
+		map<int, int> m;
```

Comment:
You can initialize this map on one line:

```cpp
map<int, int> m{{1,5},{2,9},{3,7},{4,3}};
```

and sets too

Reply:
did the module cover list initialization?

Reply:
> You can initialize this map on one line:
> 
> ```c++
> map<int, int> m{{1,5},{2,9},{3,7},{4,3}};
> ```
> 
> and sets too

hmm but i think the way i did it will be more helpful cuz u emphasized on insertions and deletions.

Reply:
At least do this for the third question? (About sets)

Reply:
w8 if we do initialization, will it still output the same answer?

Reply:
Why don’t you try and see?